examples/hello-world/step-by-step/higgs/stats/tabular_stats.ipynb

```diff
@@ -0,0 +1,918 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ccf1c433-ba7f-4967-b066-670eb5e016ed",
+   "metadata": {},
+   "source": [
+    "# Tabular Data Federated Statistics \n",
+    "\n",
+    "Before we perform machine learning tasks on the Higgs data, let's examine the statistics of the dataset. \n",
+    "\n",
+    "\n",
+    "## Setup NVFLARE\n",
+    "\n",
+    "Follow [Getting Started](https://nvflare.readthedocs.io/en/main/getting_started.html) to set up a virtual environment and install NVFLARE.\n",
+    "\n",
+    "You can also follow this [notebook](../../nvflare_setup.ipynb) to get set up.\n",
```

Comment:
need to update link for new folder structure (same in cifar10/image_stats.ipynb)